internal/pkg/utils/etcdhelper/dump.go

```diff
@@ -28,6 +29,9 @@ func DumpAll(ctx context.Context, client etcd.KV) (string, error) {
 
 		// Dump key
 		b.Write(kv.Key)
+		if kv.Lease > 0 {
+			b.WriteString(fmt.Sprintf(" (lease=%d)", kv.Lease))
+		}
```

Comment:
Aby som vedel overit, ze nejaky kluc ma nastavenu `lease` a teda, ze sa po odpojeni klienta zmaze, ... tak to pridavam do dump-u.